lobby-db/src/main/resources/db/migration/V1.07__rename_bad_words_table.sql

```diff
@@ -0,0 +1,4 @@
+create table bad_word as select word, now() as date_created from bad_words;
+alter table bad_word add primary key (word);
+alter table bad_word alter column date_created set not null;
+alter table bad_word alter column date_created set default now();
```

Comment:
Why aren't you just doing `ALTER TABLE bad_words RENAME TO bad_word`?
And then `ALTER TABLE bad_word ADD COLUMN date_created timestamptz NOT NULL DEFAULT NOW()`
Also I think we used the `COMMENT ON` feature to document what each column does in the db. Not sure if that's still consistently as available.